services/web-app/data/libraries.js

```diff
@@ -152,10 +149,10 @@ const prodLibraries = {
   //   sponsor: 'carbon'
   // },
   // 'carbon-components': {
```

Comment:
For this, the path should be `/packages/components` and the ref should be `v10` and that should allow us to index this library in the v10 branch instead of the default branch (main).